libevent-sys/wrapper.h

```diff
@@ -1,5 +1,15 @@
 #include <event.h>
 
+#include <event2/event.h>       // The primary libevent header
+#include <event2/thread.h>      // Functions for use by multithreaded programs
```

Comment:
Threading support is already included in the wrapper:
https://github.com/jmagnuson/libevent-rs/blob/27cc8f1f064c88125d6578b2b426c135308dd54d/libevent-sys/wrapper.h#L7-L9

and should only require the `threading` cargo feature flag, which is enabled by default.